### ISSUE TITLE
429 errors unreported in timeline

### ISSUE THREAD

**Issue by zigdon** [tuskyapp/Tusky#3818]
In my notifications tab, if i try to refresh and the server returns 429, i can see an error message (see screenshot).

![Screenshot_20230707-161532.png](https://github.com/tuskyapp/Tusky/assets/44372/dac3614e-a4fe-4150-8009-d4fa97b024c7)

At the same time, when i tap 'load more' in my timeline, or try to refresh, it just silently fails. 

For 'load more' at least it's obvious the call failed (since it doesn't disappear), but when refreshing there's no way to tell if there was just nothing to fetch or if the fetch failed.

The same error message should show up in the timeline tab.

**Comment by connyduck** [comment]
Yes timelines should definitely have better error reporting as well. I believe @nikclayton is already working on that.

**Comment by nikclayton** [comment]
Yes, that's coming as part of https://github.com/tuskyapp/Tusky/pull/3436
